appveyor.yml

```diff
@@ -99,6 +99,7 @@ test_script:
       --flag cardano-sl:with-wallet
       --flag cardano-sl-core:-asserts
       --flag cardano-sl-core:-dev-mode
+      --flag cardano-sl:bench
```

Comment:
But, the advantage of building benchmarks on Windows is that we will ensure they still compile. So if they don't harm much for Windows, why not enable them?
Anyway, our main bottleneck is Travis build times, not appveyor (in 99% cases appveyor finishes before).